### PR TITLE
Holmanb/doc

### PR DESCRIPTION
**Pull request by holmanb**
```
docs: fix links, add more

- several links I previously added were broken
- some links needed better names
- all links needed categorization
- line lengths in this list should not break things
```

### CODE REVIEW

File: doc/rtd/topics/faq.rst [hunk 1]
```diff
@@ -317,33 +317,51 @@ Where can I learn more?
...
 .. _cloud-init - The Good Parts: https://www.youtube.com/watch?v=2_m6EUo6VOI
-.. _cloud-init Summit 2019: https://powersj.io/post/cloud-init-summit19/
 .. _Utilising cloud-init on Microsoft Azure (Whitepaper): https://ubuntu.com/engage/azure-cloud-init-whitepaper
 .. _Cloud Instance Initialization with cloud-init (Whitepaper): https://ubuntu.com/blog/cloud-instance-initialisation-with-cloud-init
-.. _cloud-init Summit 2018: https://powersj.io/post/cloud-init-summit18/
+
 .. _cloud-init - The cross-cloud Magic Sauce (PDF): https://events.linuxfoundation.org/wp-content/uploads/2017/12/cloud-init-The-cross-cloud-Magic-Sauce-Scott-Moser-Chad-Smith-Canonical.pdf
-.. _cloud-init Summit 2017: https://powersj.io/post/cloud-init-summit17/
 .. _cloud-init - Building clouds one Linux box at a time (Video): https://www.youtube.com/watch?v=1joQfUZQcPg
 .. _cloud-init - Building clouds one Linux box at a time (PDF): https://annex.debconf.org/debconf-share/debconf17/slides/164-cloud-init_Building_clouds_one_Linux_box_at_a_time.pdf
```
Comment (aciba90) [inline doc/rtd/topics/faq.rst:356]: `annex.debconf.org` is [down](https://www.mail-archive.com/debconf-team@lists.debconf.org/msg14555.html). I have tried to find where they moved the slides without success. What do you think about offering a Wayback-Machine link?

```suggestion
.. _cloud-init - Building clouds one Linux box at a time (PDF): https://web.archive.org/web/20181111020605/https://annex.debconf.org/debconf-share/debconf17/slides/164-cloud-init_Building_clouds_one_Linux_box_at_a_time.pdf
```

File: doc/rtd/topics/faq.rst [hunk 2]
```diff
@@ -317,33 +317,51 @@ Where can I learn more?
...
 .. _Metadata and cloud-init: https://www.youtube.com/watch?v=RHVhIWifVqU
 .. _The beauty of cloud-init: http://brandon.fuller.name/archives/2011/05/02/06.40.57/
```
Comment (aciba90) [inline doc/rtd/topics/faq.rst:358]: Similarly:

```suggestion
.. _The beauty of cloud-init: https://web.archive.org/web/20180830161317/http://brandon.fuller.name/archives/2011/05/02/06.40.57/
```

**Review by aciba90 — APPROVED** [review]
Thanks, @holmanb, for the new links and categorization. It looks much better now! I have seen that a couple of links are down and left you suggestions to use archived links, if you think it could be a solution.

File: doc/rtd/topics/faq.rst (hunk 1 above)
Comment (holmanb) [reply doc/rtd/topics/faq.rst:356]: @aciba90 Good catch, thanks! It looks like sphinx has a built-in link checker. I don't think we want in CI, but I'll add a tox target in a followup PR for periodic scrubbing.

**Comment by holmanb** [comment]
Depends on: https://github.com/canonical/pycloudlib/pull/198

**Comment by holmanb** [comment]
I just rebased to drop the pycloudlib dep workaround commits I was testing with.

**Pull request merged**